src/ui/components/Notifications.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 @paritytech/contracts-ui authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { BellIcon, XIcon } from '@heroicons/react/outline';
+import { NotificationIcon } from './common/NotificationIcon';
+import type { QueuedTxOptions, NotificationsState, NotificationObject, VoidFn } from 'types';
+import { classes, isEmptyObj, truncate } from 'ui/util';
+
+interface NotificationProps {
+  dataCy?: string;
```

Comment:
`data-cy` is just another data attribute, part of HTMLAttributes. let's not clutter component props